src/script/miniscript.h

```diff
@@ -479,11 +483,16 @@ struct SatInfo {
     static constexpr SatInfo OP_VERIFY() noexcept { return {1, 1}; }
 };
 
-struct StackSize {
-    const SatInfo sat, dsat;
+class StackSize
+{
+    SatInfo sat, dsat;
 
+public:
     constexpr StackSize(SatInfo in_sat, SatInfo in_dsat) noexcept : sat(in_sat), dsat(in_dsat) {};
     constexpr StackSize(SatInfo in_both) noexcept : sat(in_both), dsat(in_both) {};
+
+    const SatInfo& Sat() const { return sat; }
```

Comment:
nit: unless there's a good reason, I'd unify the `Get`/`Is` prefixes (I actually prefer without anything here) and consider `noexcept` for these as well